openapi/src/main/java/io/helidon/openapi/SEOpenAPISupportBuilder.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.openapi;
+
+import java.util.Objects;
+
+import io.helidon.config.Config;
+import io.helidon.openapi.internal.OpenAPIConfigImpl;
+
+import io.smallrye.openapi.api.OpenApiConfig;
+import org.jboss.jandex.IndexView;
+
+/**
+ * Builds {@link OpenAPISupport} in a Helidon SE environment.
+ * <p>
+ * The builder mostly delegates to an instance of Helidon's
+ * {@link OpenAPIConfigImpl.Builder} which in turn prepares a smallrye
+ * {@link OpenApiConfig} which is what the smallrye implementation uses to
+ * control its behavior.
+ */
+public final class SEOpenAPISupportBuilder extends OpenAPISupport.Builder {
+
+    private static final String CONFIG_PREFIX = "openapi";
+    private final OpenAPIConfigImpl.Builder apiConfigBuilder = OpenAPIConfigImpl.builder();
+
+    /**
+     * Set various builder attributes from the specified {@code Config} object.
+     * <p>
+     * The {@code Config} object can specify {@value #CONFIG_PREFIX}.web-context
+     * and {@value #CONFIG_PREFIX}.static-file in addition to settings
+     * supported by {@link OpenAPIConfigImpl.Builder}.
+     *
+     * @param config the {@code Config} object possibly containing settings
+     * @return updated builder instance
+     */
```

Comment:
`NullPointerException` can be thrown from this method.  If that's by design, please add an `@exception` tag.

Reply:
True, if `config` is passed in as null, which "should never happen." Fixed.